playbooks/openshift-logging/private/config.yml

```diff
@@ -13,33 +13,93 @@
           status: "In Progress"
           start: "{{ lookup('pipe', 'date +%Y%m%d%H%M%SZ') }}"
 
+- name: Verify and collect ES hosts
+  hosts: oo_first_master
+  gather_facts: false
+  tasks:
+  - assert:
+      that: openshift_logging_es_nodeselector is defined
+      msg: "A node selector is required for Elasticsearch pods, please specify one with openshift_logging_es_nodeselector"
+
+  - name: Ensure that ElasticSearch has nodes to run on
+    import_role:
+      name: openshift_control_plane
+      tasks_from: ensure_nodes_matching_selector.yml
+    vars:
+      openshift_master_ensure_nodes_selector: "{{ openshift_logging_es_nodeselector | map_to_pairs }}"
+      openshift_master_ensure_nodes_service: Elasticsearch
+
+  - command: >
+      oc get nodes -l {{ openshift_logging_es_nodeselector | map_to_pairs }} -o jsonpath={.items[*].metadata.name}
+    register: openshift_logging_es_hosts
+
+  - when: openshift_logging_use_ops | default(false) | bool
```

Comment:
Do we need ops for 3.11/master branch?

Reply:
Currently we haven't deprecated the use of that variable, so for the time being, yes.